### PR TITLE
refresh out of date features in Overview.md

### PR DESCRIPTION
**Pull request by miguelraz**
addresses #179

### CODE REVIEW

File: training-slides/src/overview.md [hunk 1]
```diff
@@ -29,10 +29,11 @@ Rust is an empathic systems programming language that is determined to not let y
 
 ## Focus
 
--   Rust has lost many features from 2012 to 2014
+-   Rust has lost many features from since its inception:
```
Comment (jonathanpallant) [inline training-slides/src/overview.md:32]: I think this slide is specifically talking about 1.0.

Like, once upon a time Rust was weird and experimental, but then it hit 1.0 and got a proper job and a house in the suburbs.

Comment (plaindocs) [reply training-slides/src/overview.md:32]: Riight. So we're saying that tightened scope is good actually. :-D

Then I think adding a "before 1.0 release" of "before stabilizing" might help.

Comment (plaindocs) [reply training-slides/src/overview.md:32]: Because folks won't know the significance of the magic dates

Comment (jonathanpallant) [inline training-slides/src/overview.md:32]:
```suggestion
-   Rust has lost many features leading up to 1.0:
```

**Review by jonathanpallant — APPROVED** [review]

**Pull request merged**